packages/app/src/app/pages/Sandbox/Editor/Content/Preview/index.tsx

```diff
@@ -9,198 +11,249 @@ import getTemplate from '@codesandbox/common/lib/templates';
 type Props = {
```

Comment:
Can we change this to an interface? Following the convention this would be `IPreviewProps`

Reply:
Changed. Sorry that I missed it out.